portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -38,7 +38,7 @@ public double getSentimentScore() {
     return sentimentScore;
   }
 
-  public double getTimestamp() {
-    return timestamp;
+  public double getTimestamp_ms() {
```

Comment:
nit: getTimestampMs, function names never have an underscore or dash in Java (or JS)